demo/dynamic_embedding/movielens-1m-keras-with-horovod/movielens-1m-keras-with-horovod.py

```diff
@@ -0,0 +1,242 @@
+import os
```

Comment:
It's recommended to provide a README.md for this excellent demo.

Reply:
Done.